app/routes/events/components/EventList.tsx

```diff
@@ -217,58 +223,66 @@ const EventList = () => {
     };
 
   return (
-    <div className={styles.root}>
+    <Page
+      title="Arrangementer"
+      sidebar={{
+        title: 'Filter',
+        icon: 'filter',
+        side: 'right',
+        content: (
+          <Flex column gap={8}>
+            <h4>Arrangementtype</h4>
+            <Flex column>
+              <CheckBox
+                id="companyPresentation"
+                label="Bedpres"
+                checked={showCompanyPresentation}
+                onChange={toggleEventType('company_presentation')}
+              />
+              <CheckBox
+                id="course"
+                label="Kurs"
+                checked={showCourse}
+                onChange={toggleEventType('course')}
+              />
+              <CheckBox
+                id="social"
+                label="Sosialt"
+                checked={showSocial}
+                onChange={toggleEventType('social')}
+              />
+              <CheckBox
+                id="other"
+                label="Annet"
+                checked={showOther}
+                onChange={toggleEventType('other')}
+              />
+            </Flex>
+            <h4>Påmelding</h4>
```

Comment:
Would be nice with a little bit more margin between the two filters. Right now, the four elements (2x header, 1x checkbox-group and 1x dropdown) all have the same spacing, which feels off.

Suggest to double the spacing between the checkboxes and "Påmelding"

Reply:
Agreed, but I didn't want to spend too much time styling these in this PR. I will create a shared component for filter-menus later, so they all have consistent styling.

Reply:
Maybe I'll just do it here, it wouldn't grow the PR that much